### ISSUE TITLE
exit the action immediately after saying AQI for returning users

### ISSUE THREAD

**Issue by birdowl** [codeminders/violet-air#21]
it is a bit annoying that the action stays after saying AQI. Especially if you call it multiple times per day. Having to call "OG, close" is very annoying. At the same time, we want to offer the user a chance to update the stored location or change smoke correction settings. 

I propose we stay in the action for the first time user but exit for returning users. Also, for the returning users, in addition to the message 'You can ask me to update the location or turn off the smoke correction.', we can say "just say "OK Google, ask Purple Turtle to update location". We should not say this each time, but maybe every 5th time or so. I do not want the action to be too wordy.

@alexsaveliev what do you think?

**Comment by alexsaveliev** [comment]
since we don't have canvas anymore, I'd close it every time, adding, if necessary, hints what user can do

**Comment by birdowl** [comment]
@alexsaveliev then do we need suggestion chips?
